tools/test.py

```diff
@@ -1427,6 +1427,9 @@ def BuildOptions():
   result.add_option('--abort-on-timeout',
       help='Send SIGABRT instead of SIGTERM to kill processes that time out',
       default=False, action="store_true", dest="abort_on_timeout")
+  result.add_option("--type",
+      help="Specifies the type of build. Options: fips"
```

Comment:
Nit (non-blocking): Can we format this more like existing options?

For example, using this as a model:

> "The style of progress indicator (verbose, dots, color, mono, tap)"

...this would be:

> "Type of build (default, fips)"

...or don't list the options at all if they're not enumerated by this file (which I don't see where they are).

Reply:
The options are default or fips AFAIK.

Reply:
Yeah, the options are `default` or `fips`.

You don't see the enumeration as there are no more choices and the existing code parses the `type = fips` to give you PASS, FAIL, SKIP functionality. It took me some time to figure this out, code is crazy old, pretty complicated to read and I do intend to follow up with a blog entry on this soon(ish), for knowledge sharing.

Reply:
... also this will have to change a bit for `shared` case when we might see some enumertaion

Reply:
@Trott (formatting has been done in latest)